src/components/common/progress/ProgressIndicator.tsx

```diff
@@ -0,0 +1,25 @@
+import ProgressBridge from './ProgressBridge';
+import ProgressIndex from './ProgressIndex';
+
+interface ProgressIndicatorProps {
+  totalStep: number;
+  currentStep: number;
+}
+
+function ProgressIndicator({ totalStep, currentStep }: ProgressIndicatorProps) {
+  console.log(new Array(totalStep).fill(0));
+  return (
+    <div className='flex'>
+      <ProgressIndex isActive={true}>{1}</ProgressIndex>
+      {totalStep > 1 &&
+        new Array(totalStep - 1).fill(0).map((_, index) => (
```

Comment:
(가독성 측면)
```suggestion
      <ProgressIndex isActive={true}>{1}</ProgressIndex>
      {totalStep > 1 &&
         Array.from({ length: totalStep - 1 }).map((_, index) => (
```

기존 방식도 좋아보여요~

Reply:
이렇게 두고 비교해보니 length란 이름도 있고 new 연산자도 안적어도 되서 수정해주신 부분이 가독성 좋아보이는 것 같습니다! 👍👍 의견 감사합니다!

Reply:
저도 이후 로직에 `+index`를 하는 경우에 undefined+index 값이 잘 적용되는지 테스트를 해보긴 했는데, 문제없이 적용되는 거 같더라구요! 

하나 차이는 전부 0으로 초기화 / 전부 undefined로 초기화(빈배열) 이라고 생각하시면 될 거 같아요~